internal/release/github.go

```diff
@@ -0,0 +1,171 @@
+package release
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"aead.dev/minisign"
+	"github.com/google/go-github/v48/github"
+	"github.com/hashicorp/go-retryablehttp"
+	"golang.org/x/oauth2"
+)
+
+type GithubRepo string
+type GithubOwner string
```

Comment:
Not strictly necessary but I want to avoid passing the wrong thing to the client methods (easy to get things out of order).